echopop/nwfsc_feat/spatial.py

```diff
@@ -0,0 +1,639 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from lmfit import Minimizer, Parameters
+from scipy import interpolate
+
+from ..spatial.variogram import variogram
+
+
+def standardize_coordinates(
+    data_df: pd.DataFrame,
+    longitude_offset: float = 0.0,
+    latitude_offset: float = 0.0,
+    reference_df: Optional[pd.DataFrame] = None,
```

Comment:
Suggest changing to `df_data` and `df_reference` to be consistent with other variable naming.